src/main/kotlin/snyk/common/SnykCachedResults.kt

```diff
@@ -145,21 +152,49 @@ class SnykCachedResults(val project: Project) : Disposable {
                 }
 
                 override fun scanningError(snykScan: SnykScanParams) {
+                    // TODO: get actual error message from LS
+                    when (snykScan.product) {
+                        "oss" -> {
+                            currentOssError =
+                                SnykError("Language Server failed to run Snyk OpenSource scan", snykScan.folderPath)
```

Comment:
Should we expose the detail, that Language Server runs the scan? I'm biased, but it may be information the customer does not need.

Reply:
Good point, I'll update this 👍